doc/api/deprecations.md

```diff
@@ -3019,6 +3019,17 @@ should have the same effect. The receiving end should also check the
 [`readable.readableEnded`][] value on [`http.IncomingMessage`][] to get whether
 it was an aborted or graceful destroy.
 
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/40773
+    description: Documentation-only deprecation.
+-->
+
+Type: Runtime
+
+Streams no longer support thenables in implementation methods.
```

Comment:
Can we guide users to another stream feature that would achieve the same as a thenable?
```suggestion
An undocumented feature of Node.js streams was to support thenables in implementation methods. This is now deprecated, instead use ???
```